src/main/java/com/puppycrawl/tools/checkstyle/checks/whitespace/EmptyForIteratorPadCheck.java

```diff
@@ -51,6 +51,15 @@
  * &lt;module name=&quot;EmptyForIteratorPad&quot;/&gt;
  * </pre>
  * <p>
+ * Example:
+ * </p>
+ * <pre>
+ * for (Iterator foo =mp.entrySet().iterator();  foo.hasNext(); ); // violation since whitespace
+ *                                                                 //after semicolon
```

Comment:
```
 * for (Iterator it = map.entrySet().iterator();  it.hasNext(); ); // violation since whitespace
 *                                                                 // after semicolon
```

Reply:
apply the same for all other cases.

Reply:
not done, 
https://checkstyle-diff-reports.s3.us-east-2.amazonaws.com/emptyforiteratorpad_2021033434/config_whitespace.html#EmptyForIteratorPad

there is not space after `=` and trialing comments are not aligned.

Reply:
please also make this example but without violation

Reply:
@romani done.